src/drivers/github.js

```diff
@@ -620,33 +620,26 @@ class Github {
   async job(opts = {}) {
     const { time, runnerId } = opts;
     const { owner, repo } = ownerRepo({ uri: this.repo });
-    const { actions } = octokit(this.token, this.repo);
+    const octokitClient = octokit(this.token, this.repo);
 
     let { status = 'queued' } = opts;
     if (status === 'running') status = 'in_progress';
 
-    const {
-      data: { workflow_runs: workflowRuns }
-    } = await actions.listWorkflowRunsForRepo({
-      owner,
-      repo
-    });
+    const workflowRuns = await octokitClient.paginate(
+      octokitClient.actions.listWorkflowRunsForRepo,
+      { owner, repo }
+    );
 
     let runJobs = await Promise.all(
       workflowRuns
         .filter(({ status: jobStatus }) => jobStatus === status)
-        .map(async (run) => {
-          const {
-            data: { jobs }
-          } = await actions.listJobsForWorkflowRun({
-            owner,
-            repo,
-            run_id: run.id,
-            status
-          });
-
-          return jobs;
-        })
+        .map(
+          async ({ id }) =>
```

Comment:
I'm guessing the `run/run.id -> id` is this: https://discord.com/channels/485586884165107732/728693131557732403/963472006651736074

Reply:
No, that mistake was that I missed some parentheses here:
https://github.com/iterative/cml/blob/d70f5ebb4d403d57fa7d13936a5aa6de6f7ffba8/src/drivers/github.js#L637

`({ status, id }) => { status, id }` should be `({ status, id }) => ({ status, id })`

Reply:
I see, I miss read the object un-packing 🙈